### ISSUE TITLE
Use of numpy.random in BiasedRandomWalk is unseeded

### ISSUE THREAD

**Issue by huonw** [stellargraph/stellargraph#235]
**Describe the bug**

`BiasedRandomWalk` uses `numpy.random`, but the seed arg to `__init__` and to `run` are only fed into `random.seed` (the built-in one) not `numpy.random.seed`.

**To Reproduce**

Multiple calls to `BiasedRandomWalk(..., seed=1)` will give different sequences for identical calls to `run`, as will multiple calls to `run(..., seed=1)`. The following will print `False`.

```python
import stellargraph as sg
import stellargraph.data.explorer as sge
import networkx as nx

graph = nx.Graph()
graph.add_edges_from([
    (0, 1),
    (0, 2),
])
# 0-1
# |
# 2

graph = sg.StellarGraph(graph)
walker = sge.BiasedRandomWalk(graph, seed=1)
run1 = walker.run([0], 10, length=2)

walker2 = sge.BiasedRandomWalk(graph, seed=1)
run2 = walker2.run([0], 10, length=2)

print(run1 == run2)
```

https://colab.research.google.com/drive/19pWRvkI_pAU-KnPbv9qghVFY0Hkfv1YJ

**Expected behavior**

The above should print `True`.

**Screenshots**

NA.

**Desktop (please complete the following information):**

Stellargraph 0.4.0.

**Comment by adocherty** [comment]
Hi .. Thanks for the bugreport! 🐞 

I think I sneaked in a fix to this in the `develop` branch with the latest PR #232. I've tried your code and it now seems to work.

**Comment by huonw** [comment]
This was fixed in #232 (and also, partially, in #236).
